addons/hr_timesheet/models/project.py

```diff
@@ -269,6 +269,7 @@ class Task(models.Model):
     total_hours_spent = fields.Float("Total Hours", compute='_compute_total_hours_spent', store=True, help="Time spent on this task, including its sub-tasks.")
     progress = fields.Float("Progress", compute='_compute_progress_hours', store=True, group_operator="avg", help="Display progress of current task.")
     overtime = fields.Float(compute='_compute_progress_hours', store=True)
+    soon_overtime = fields.Float(compute='_compute_soon_overtime_hours', store=True, default=0.0)
```

Comment:
I update my previous suggestion, since I was wrong.
```suggestion
    remaining_hours_percentage = fields.Float(compute='_compute_remaining_hours_percentage', search='_search_remaining_hours_percentage')
```
Also, can you move this field below the `remaining_hours` field? Thanks in advance!